query.js

```diff
@@ -159,7 +160,7 @@ exports.init = function (sbot, config) {
         subfeed,
         keys,
         metadata,
-        seed: feedformat === 'bendy butt' ? seed : undefined,
+        seed: feedformat === 'bendybutt-v1' ? seed : undefined,
```

Comment:
This check should be `type === 'metafeed/add/existing'`

Reply:
It seems our tests does not catch this. Maybe add a test where you add an existing feed & derived feed and later restore from a seed.

Reply:
I can have a go at fixing this

Reply:
https://github.com/ssb-ngi-pointer/ssb-meta-feeds/pull/33